src/common/IndividualBuildConfig.groovy

```diff
@@ -123,6 +124,7 @@ class IndividualBuildConfig implements Serializable {
         RELEASE = map.get("RELEASE")
         PUBLISH_NAME = map.get("PUBLISH_NAME") != null ? map.get("PUBLISH_NAME").trim() : null
         ADOPT_BUILD_NUMBER = map.get("ADOPT_BUILD_NUMBER") != null ? map.get("ADOPT_BUILD_NUMBER").trim() : null
+        ENABLE_REPRODUCIBLE_COMPARE = map.get("ENABLE_REPRODUCIBLE_COMPARE")
```

Comment:
Does this need a check for not null, like lines above? if invoked with an old config not containing it?

Reply:
Correct, updated.